src/routes/webhook/webhook.service.ts

```diff
@@ -86,26 +89,52 @@ export class WebhookService {
   ): Promise<AxiosResponse | undefined> {
     const headers = authorization ? { Authorization: authorization } : {};
     const startTime = Date.now();
-    const strMessage = JSON.stringify(parsedMessage);
     return firstValueFrom(
       this.httpService.post(url, parsedMessage, { headers }).pipe(
         catchError((error: AxiosError) => {
           if (error.response !== undefined) {
             // Response received status code but status code not 2xx
             const responseData = this.parseResponseData(error.response.data);
-            this.logger.error(
-              `Error sending event ${strMessage} to ${url}: ${error.response.status} ${error.response.statusText} - ${responseData}`,
-            );
+            this.logger.error({
+              message: 'Error sending event',
+              messageContext: {
+                event: parsedMessage,
+                httpRequest: {
+                  url: url,
+                  startTime: startTime,
+                },
+                httpResponse: {
+                  data: responseData,
+                  statusCode: error.response.status,
+                },
+              },
+            });
           } else if (error.request !== undefined) {
             // Request was made but response was not received
-            this.logger.error(
-              `Error sending event ${strMessage} to ${url}: Response not received. Error: ${error.message}`,
-            );
+            this.logger.error({
+              message: `Error sending event: Response not received. Error: ${error.message}`,
+              messageContext: {
+                event: parsedMessage,
+                httpRequest: {
+                  url: url,
+                  startTime: startTime,
+                },
+                httpResponse: null,
```

Comment:
Should this be null or can it be ignored?

Reply:
I set it to null to make clear that it was a http request but there was no response